packages/core/strapi/bin/strapi.js

```diff
@@ -255,4 +257,157 @@ program
   .option('-s, --silent', `Run the generation silently, without any output`, false)
   .action(getLocalScript('ts/generate-types'));
 
+// `$ strapi export`
+const parseBool = (arg) => {
```

Comment:
Should this live here?

Reply:
I can see it living here since it's just a little helper to cast inputs, but if you think it would be better somewhere else I would be happy to move it to either:
1) a CLI-specific helpers module (which we don't currently have)
2) `@strapi/utils/lib/parseType` -- would be my preference _except_ that it goes against the way the other methods work (converting errors to `false`) so I don't think we should

Your thoughts?

Reply:
On second thought, maybe this should also throw (and abort the command). I'm going to change that. But we still need somewhere for the helper.